cfme/utils/ssh.py

```diff
@@ -842,3 +849,35 @@ def keygen():
     with _ssh_pubkey_file.open('w') as f:
         f.write("{} {} {}\n".format(pub.get_name(), pub.get_base64(),
             'autogenerated cfme_tests key'))
+
+
+def unquirked_ssh_client(hostname, *args, **kwargs):
+    """ A factory for the SSHClient dealing with some of its quirks. """
+    if hostname is None:
+        raise ValueError(f'Hostname cannot be {hostname}')
+    client = SSHClient(hostname=hostname, *args, use_check_port=False, **kwargs)
```

Comment:
SSHClient doesn't accept any positional arguments, so `*args` isn't needed here or in the definition of unquirked_ssh_client.

```
class SSHClient(paramiko.SSHClient):

    def __init__(self, stream_output=False, **connect_kwargs):
```

Reply:
I didn't check that. I though it may be good to stay on the safe side. It doesn't hurt anything except perhaps someone may expect it is there for specific purpose.